README.md

```diff
@@ -0,0 +1,55 @@
+# in-toto Layout Creation Wizard
+
+A Flask based web app to guide project owners through creating an
+[in-toto layout](https://in-toto.io).
```

Comment:
The linked text is `in-toto layout`, so I was expecting a link to something about in-toto's layout tool.

Maybe linking to https://in-toto.engineering.nyu.edu/, where an instance of the layout tool exists, makes more sense?  This repository is also more about the layout tool rather than in-toto.

Reply:
Done